src/test/java/com/google/devtools/build/lib/buildtool/QueryIntegrationTest.java

```diff
@@ -340,6 +340,35 @@ public void testInputFileElementContainsPackageGroups() throws Exception {
         .isNotNull();
   }
 
+  @Test
+  public void testExecutablesQuery() throws Exception {
```

Comment:
I don't fully understand the query test inheritance hierarchy - does this single test cover both cquery and aquery? What about query? Being executable is an unconfigured property.

Reply:
fixed so that the answer to this is now yes, all 3 queries are tested.